.tool/lint

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
I'm not sure whether the cri-o pattern is intended. To me, if makes more sense to move them all into `hack/` if there is no particular reason that we should separate them.